src/applications/vaos/tests/e2e/fixtures/MockAppointmentResponse.js

```diff
@@ -92,10 +93,16 @@ export default class MockAppointmentResponse {
       },
       future,
       pending,
+      past,
     };
   }
 
-  static createAtlasResponses({ localStartTime, future = false, count = 1 }) {
+  static createAtlasResponses({
+    localStartTime,
+    future = false,
+    past = false,
+    count = 1,
+  }) {
```

Comment:
nit: I don’t think you need to include "past=false" as a parameter for `createAtlasResponses`, `createClinicResponses` and`createGfeResponses`, since they all rely on `MockAppointmentResponse` where "past=false" is the default.